### PR TITLE
V0.2

### PR DESCRIPTION
**Pull request by Godd-Howard**

### CODE REVIEW

File: Journal.md [hunk 1]
```diff
@@ -21,3 +21,8 @@ Camryn worked on the Layout for the webpage
 ### 2/20/20, 3:00pm - Added variable typing
 Dear Diary,
 Jonathan worked on the adding variable types for the different variables that we have.  He made sure to have the interpreter not be able to change variable types and added in auto widening and narrowing so that integers and reals can be changed around.
+
+### 3/1/2020, 1:00am - Added menu, links, stylized layout, lined input
+Kurtis added a menu with links to other pages on the site, including, Documentation, Examples, and Help. He also stylized some elements of the layout.
+He added a toggle button for a light/dark mode. He lined the input box and added a vertical scroll bar. He put added a examples from the CS 101 book to the examples page.
```
Comment (jimender2) [inline Journal.md]: Fix the grammar here.

Comment (Godd-Howard) [reply Journal.md]: Yes, sir.

**Comment by jimender2** [comment]
So some small issues right now that I see @Godd-Howard.

When you toggle darkmode, the logo goes away when you go to dark mode.
Try to find another place to put the menu button, and the dark mode toggle. A better place for the menu button would be next to or under the logo. Making that whole top area compact is a good thing.
Have the input and output box fonts be the same size font and everything.

I do like the outlines around the input/output boxes. Nice touch.

**Comment by Godd-Howard** [comment]
Getting rid of the logo is planned not a bug. You would have to go into the
image file and sift through the Madman image to be able to be able to turn
it white because it looks dumb otherwise with basically black on black if
left alone.

I'll move the logo that makes sense.

What do you mean by the input and output fonts?

**Comment by jimender2** [comment]
image
If you look at the photo, you can see that where I put Display "test" looks different from where it actually displayed the test.

It looks like it might be leftover from the "Output" header that you have above the output area. A custom css class that is applied to both the "console" and "code" divs should fix that issue.

**Pull request merged**